docker/setup.sh

```diff
@@ -4,5 +4,5 @@
 git clone -b master https://github.com/hysds/multi_acquisition_localizer.git
 
 # clone ariamh to be moved to its final location by docker builder
-git clone -b standard-product https://github.com/hysds/ariamh.git
+git clone -b python3 https://github.com/hysds/ariamh.git
 
```

Comment:
To resolve the issue where import isce fails as PATHs to isce in conda are not set right via isce.sh when using hysds/isce2 as per: aria-jpl/ariamh@fbd11b6#diff-2da18fa62d05690f76a9a362bd9c3d84R6